src/uct/api/v2/uct_v2.h

```diff
@@ -549,43 +555,49 @@ ucs_status_t uct_md_mem_dereg_v2(uct_md_h md,
  */
 typedef enum uct_md_attr_field {
     /** Indicate max allocation size. */
-    UCT_MD_ATTR_FIELD_MAX_ALLOC        = UCS_BIT(0),
+    UCT_MD_ATTR_FIELD_MAX_ALLOC                 = UCS_BIT(0),
 
     /** Indicate max registration size. */
-    UCT_MD_ATTR_FIELD_MAX_REG          = UCS_BIT(1),
+    UCT_MD_ATTR_FIELD_MAX_REG                   = UCS_BIT(1),
 
     /** Indicate capability flags. */
-    UCT_MD_ATTR_FIELD_FLAGS            = UCS_BIT(2),
+    UCT_MD_ATTR_FIELD_FLAGS                     = UCS_BIT(2),
 
     /** Indicate memory types that the MD can register. */
-    UCT_MD_ATTR_FIELD_REG_MEM_TYPES    = UCS_BIT(3),
+    UCT_MD_ATTR_FIELD_REG_MEM_TYPES             = UCS_BIT(3),
 
     /** Indicate memory types that the MD can register. */
-    UCT_MD_ATTR_FIELD_CACHE_MEM_TYPES  = UCS_BIT(4),
+    UCT_MD_ATTR_FIELD_CACHE_MEM_TYPES           = UCS_BIT(4),
```

Comment:
duplicate comment of UCT_MD_ATTR_FIELD_REG_MEM_TYPES

Reply:
fixed